common/djangoapps/edxmako/shortcuts.py

```diff
@@ -90,7 +90,10 @@ def marketing_link(name):
             if all([host_name and 'edge' in host_name, 'http' in link_map[name]]):
```

Comment:
This existing line is odd. Isn't it just this?

```python
host_name and 'edge' in host_name and 'http' in link_map[name]
```

Given that the error we're seeing is `'https' is not a registered namespace`, the only way we get down to line 93 is if the hostname is empty or doesn't contain edge, right?